### PR TITLE
Add multi-website features to Email templates 

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

**Comment by TimLai125** [comment]
@yostashiro The PR is ready to review.

File: mail_multi_website_email/models/res_partner_hook_get_signup_url_for_action.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    #base_url = self.env['ir.config_parameter'].get_param('web.base.url')
+    for partner in self:
+
+        """QTL MOD - Search if the user account has the multi-website
+        settings and set up the base url with correct domain
+        """
+        base_url = http.request.env[
+            'ir.config_parameter'].get_param('web.base.url')
+        domain = [
+            ('partner_id', '=', partner.id)
```
Comment (yostashiro) [inline mail_multi_website_email/models/res_partner_hook_get_signup_url_for_action.py]: No need to include `(active, '=', True)`?  I guess it is technically possible to link multiple users to a partner although it shouldn't happen under normal circumstances.

File: stock_delivery_notification/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 {
     'name': 'Delivery Order Notification',
-    'version': '10.0.1.1.3',
+    'version': '10.0.1.2.3',
```
Comment (yostashiro) [inline stock_delivery_notification/__manifest__.py]: `'10.0.1.2.0'`

File: stock_delivery_notification/models/stock_picking.py [hunk 3]
```diff
@@ -38,8 +38,16 @@ def action_send_delivery_order(self):
             if sale_order.team_id and sale_order.team_id.id == \
                     self.env.ref('sales_team.salesteam_website_sales').id:
                 self.origin_sale = sale_order.name
-                base_url = http.request.env['ir.config_parameter'].get_param(
-                    'web.base.url')
+                base_url = http.request.env[
+                    'ir.config_parameter'].get_param('web.base.url')
+                if sale_order.partner_id:
+                    domain = [
+                        ('partner_id', '=', sale_order.partner_id.id)
```
Comment (yostashiro) [inline stock_delivery_notification/models/stock_picking.py]: No need to include `('active', '=', False)`?

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by TimLai125** [comment]
@yostashiro I have changed the implementation in the sale_order, i.e. override the get_access_action method to add the domain url hence overwriting the default email template is not needed.

**Review by yostashiro — APPROVED** [review]
👍

**Pull request merged**